managed/services/inventory/grpc/services_server.go

```diff
@@ -20,21 +20,29 @@ import (
 	"fmt"
 
 	"github.com/AlekSi/pointer"
+	"github.com/pkg/errors"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
 
 	"github.com/percona/pmm/api/inventorypb"
 	"github.com/percona/pmm/managed/models"
 	"github.com/percona/pmm/managed/services/inventory"
+	"github.com/percona/pmm/managed/services/management/common"
 )
 
 type servicesServer struct {
-	s *inventory.ServicesService
+	s            *inventory.ServicesService
+	mgmtServices common.MgmtServices
 
 	inventorypb.UnimplementedServicesServer
 }
 
 // NewServicesServer returns Inventory API handler for managing Services.
-func NewServicesServer(s *inventory.ServicesService) inventorypb.ServicesServer {
-	return &servicesServer{s: s}
+func NewServicesServer(s *inventory.ServicesService, mgmtServices common.MgmtServices) inventorypb.ServicesServer {
```

Comment:
Plural of plurals is not a good language construct. It should be rather `NewServiceServer`.

Reply:
Not sure where exactly the plural of plurals, I see only one plural, but I agree it sounds a little strange. Yet we have plenty of them like `NewNodesServer`, `NewAgentsServer` etc. So I'm not sure we need to do refactoring in the scope of this pr.

Reply:
I'm not in favor of refactoring now. Rather, we have a good opportunity to do it right this time.

Reply:
Ok, I've just checked. This notation comes from grpc api, and if we just rename one func we'd get inconsistency. We can discuss it on a weekly call, but for now I don't think we really need change something here.

Reply:
It's not about renaming - we are creating a brand new function :)

Reply:
Apart from that, do we need to be consistent in poor naming?

Reply:
Actually this is an old function, we just need to add new dependency. About poor naming: let's discuss it with the team)